packages/@react-spectrum/list/src/ListView.tsx

```diff
@@ -157,53 +156,44 @@ function ListView<T extends object>(props: ListViewProps<T>, ref: DOMRef<HTMLDiv
-  let {checkboxProps} = useGridSelectionCheckbox({key: null}, state);
   let dragState: DraggableCollectionState;
   if (isListDraggable) {
     dragState = dragHooks.useDraggableCollectionState({
       collection: state.collection,
       selectionManager: state.selectionManager,
-      renderPreview(selectedKeys, draggedKey) {
+      renderPreview(draggingKeys, draggedKey) {
         let item = state.collection.getItem(draggedKey);
+        let isDraggingMultiple = draggingKeys.size > 1;
         let itemWidth = domRef.current.offsetWidth;
-        let showCheckbox = state.selectionManager.selectionMode !== 'none' && state.selectionManager.selectionBehavior === 'toggle';
-        let isSelected = state.selectionManager.isSelected(item.key);
+
         return (
           <Provider
             {...provider}
-            UNSAFE_className={classNames(listStyles, 'react-spectrum-ListViewItem', 'is-dragging')}
-            UNSAFE_style={{width: itemWidth, paddingInlineStart: 0}}>
-            <div className={listStyles['react-spectrum-ListViewItem-grid']}>
-              <div className={listStyles['react-spectrum-ListViewItem-draghandle-container']}>
-                <div className={listStyles['react-spectrum-ListViewItem-draghandle-button']}>
-                  <ListGripper />
-                </div>
-              </div>
-              {showCheckbox &&
-                <Checkbox
-                  isSelected={isSelected}
-                  UNSAFE_className={listStyles['react-spectrum-ListViewItem-checkbox']}
-                  isEmphasized
-                  aria-label={checkboxProps['aria-label']} />
-              }
-              <SlotProvider
-                slots={{
-                  content: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content']},
-                  text: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content']},
-                  description: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-description']},
-                  icon: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-icon'], size: 'M'},
-                  image: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-image']},
-                  link: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-content'], isQuiet: true},
-                  actionButton: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-actions'], isQuiet: true},
-                  actionGroup: {
-                    UNSAFE_className: listStyles['react-spectrum-ListViewItem-actions'],
-                    isQuiet: true,
-                    density: 'compact'
-                  },
-                  actionMenu: {UNSAFE_className: listStyles['react-spectrum-ListViewItem-actionmenu'], isQuiet: true}
-                }}>
-                {typeof item.rendered === 'string' ? <Content>{item.rendered}</Content> : item.rendered}
-              </SlotProvider>
+            UNSAFE_style={{width: itemWidth, direction}}>
```

Comment:
One question is whether the width should always match the width of list view or whether it should be a fixed size. If the list view is super wide for example, would we want the drag preview to also be super wide? I'm thinking probably not, or at least there should be a max. Possibly a question for spectrum.

Reply:
It does feel strange to drag a very wide item, but you run the risk of having a different preview if it is less wide (i.e. text overflows to new line), which may feel weird. 

Along the same lines, I wonder if want to scale transform the preview down any. The RFC mentions that the preview should be smaller so you can see behind it, but designs don't specify.

Reply:
Updated to use a fixed width of 194px for now, after discussing with spectrum